db/migrate/20180918135943_change_default_pxe_items.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeDefaultPxeItems < ActiveRecord::Migration[5.2]
+  def up
+    Setting.without_auditing do
+      ['default_pxe_item_global', 'default_pxe_item_local'].each do |name|
+        setting = Setting.where(:name => name).first
+        setting&.update_attribute(:value, "--- ''")
```

Comment:
Did you mean `:default` instead of `:value`?

Also, the value doesn't seem to work this way, as it leads to `default: "--- \"--- ''\"\n"`,  while you probably want `setting&.update_attribute(:value, nil)`

Reply:
Oh right, actually nil does not work, it needs to be blank value because global_setting test for that. That's the reason why this is "blankable" so users can set it back to blank. Here is full test:

```
[1] pry(main)> Setting.find_by_name(:default_pxe_item_local)
=> #<Setting::Provisioning:0x000055555ea80fd0
 id: 209,
 name: "default_pxe_item_local",
 value: nil,
 description: "Default PXE (PXELinux, PXEGrub2) menu item in local template - 'local_chain_hd0', 'local' or custom",
 category: "Setting::Provisioning",
 settings_type: "string",
 default: "--- local_chain_hd0\n...",
 created_at: Thu, 20 Sep 2018 16:13:32 UTC +00:00,
 updated_at: Thu, 20 Sep 2018 16:13:32 UTC +00:00,
 full_name: "Default PXE local template entry",
 encrypted: false>
[2] pry(main)> include Foreman::Renderer::Scope::Macros::Base
=> Object
[3] pry(main)> global_setting("default_pxe_item_local", "xxx")
=> "local_chain_hd0"
[4] pry(main)> Setting[:default_pxe_item_local]
=> "local_chain_hd0"
[5] pry(main)> ['default_pxe_item_global', 'default_pxe_item_local'].each do |name|      
[5] pry(main)*   setting = Setting.where(:name => name).first        
[5] pry(main)*   setting&.update_attribute(:default, '')        
[5] pry(main)* end        
Setting (208) update event on default --- local
 | ..., --- ''
 | 
Setting (209) update event on default --- local_chain_hd0
 | ..., --- ''
 | 
=> ["default_pxe_item_global", "default_pxe_item_local"]
[6] pry(main)> Setting.find_by_name(:default_pxe_item_local)
=> #<Setting::Provisioning:0x0000555556ea73a0
 id: 209,
 name: "default_pxe_item_local",
 value: nil,
 description: "Default PXE (PXELinux, PXEGrub2) menu item in local template - 'local_chain_hd0', 'local' or custom",
 category: "Setting::Provisioning",
 settings_type: "string",
 default: "--- ''\n",
 created_at: Thu, 20 Sep 2018 16:13:32 UTC +00:00,
 updated_at: Thu, 20 Sep 2018 16:15:49 UTC +00:00,
 full_name: "Default PXE local template entry",
 encrypted: false>
[7] pry(main)> global_setting("default_pxe_item_local", "xxx")
=> "xxx"
[8] pry(main)> Setting[:default_pxe_item_local]
=> ""
```